tests/test_model.py

```diff
@@ -389,7 +470,28 @@ def test_serialization_by_filename(self):
         for n0, n1 in zip(model.iter_symbols(), new.iter_symbols()):
             self.assertIs(type(n0), type(n1))
 
-    def test_serialization_max_num_states(self):
+    def test_invalid_version(self):
+        from dwave.optimization._model import DEFAULT_SERIALIZATION_VERSION
+
+        model = Model()
+        with io.BytesIO() as f:
+            model.into_file(f)
```

Comment:
want to test the path where you explicitly set `version` as well?

Reply:
Good call, and adding that test led me to another bug fix: https://github.com/arcondello/dwave-optimization/blob/2a4bca442b20200d0ee2937e683f217491b76b92/dwave/optimization/model.py#L418-L422